src/sentry/scripts/similarity/index.lua

```diff
@@ -631,6 +652,137 @@ local commands = {
+                    end
+                end
+            end
+        end
+    ),
+    EXPORT = takes_configuration(
+        --[[
+        Exports data that is located at the provided ``index`` and ``key`` pairs.
+
+        Generally, this data should be treated as opaque method for extracting
+        data to be provided to the ``IMPORT`` command. Exported data is
+        returned in the same order as the arguments are provided. Each item is
+        a messagepacked blob that is at the top level list, where each member
+        represents the data contained within one band.  Each item in the band
+        list is another list, where each member represents one time series
+        interval. Each item in the time series list is a tuple containing the
+        time series index and a mapping containing the counts for each bucket
+        within the interval. (Due to the Lua data model, an empty mapping will
+        be represented as an empty list. The consumer of this data must convert
+        it back to the correct type.)
+        ]]--
+        function (configuration, arguments)
+            local bands = range(1, configuration.bands)
+            local time_series = get_active_indices(
+                configuration.interval,
+                configuration.retention,
+                configuration.timestamp
+            )
+            return table.imap(
+                collect_index_key_pairs(arguments),
+                function (source)
+                    return cmsgpack.pack(
+                        table.imap(
+                            bands,
+                            function (band)
+                                return table.imap(
+                                    time_series,
+                                    function (time)
+                                        return {
+                                            time,
+                                            redis_hgetall_response_to_table(
+                                                redis.call(
+                                                    'HGETALL',
+                                                    get_bucket_frequency_key(
+                                                        configuration.scope,
+                                                        source.index,
+                                                        time,
+                                                        band,
+                                                        source.key
+                                                    )
+                                                ),
+                                                tonumber
+                                            ),
+                                        }
+                                    end
+                                )
+                            end
+                        )
+                    )
+                end
+            )
+        end
```

Comment:
writin some lisp over here except someone macro'd `end` to be another paren token